build/ci/evergreen.yml

```diff
@@ -1299,6 +1299,7 @@ tasks:
   - name: atlas_cleanup_e2e
     tags: [ "e2e","cleanup", "foliage_infrequent_task" ]
     must_have_test_results: true
+    stepback: false
```

Comment:
Supprised that this is an `true` by default :)

Reply:
https://github.com/mongodb/mongodb-atlas-cli/blob/c3724c784b4f10602cb6d90b901e86a7b7d98a4e/build/ci/evergreen.yml#L15-L18